src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,50 @@
+# Navigate Between Windows and Applications
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Navigate Between Windows
+
+View all windows in the current workspace by pressing `SUPER` + `D`. For more information about Workspaces, see [Using Workspaces](using-workspaces.md)
+
+![Display All Windows](/images/locate-launch-applications/display-all-windows.png)
+
+Use the the `🠐` `🠑` `🠒` `🠓` keys or mouse to move between windows. Select a window with left-click or by pressing `Enter`.
+
+![Select Displayed Window](/images/locate-launch-applications/select-displayed-window.png)
+
+Switch back to the previously selected window by pressing `Alt` + `Esc`.
+
+### Navigate Windows with Keyboard Shortcuts
```

Comment:
It's great that this is here, but related to an above comment, something seems off about this being a subsection of the previous one. Not sure if you'd consider making the previous section titled to refer to the Workspaces menu in particular.

Reply:
I think the content reorganization addresses this comment, but let me know if not.